common/src/main/java/com/box/l10n/mojito/DateTimeUtils.java

```diff
@@ -0,0 +1,75 @@
+package com.box.l10n.mojito;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.Date;
+
+/**
+ * Util class to provide Date/DateTime utility methods such as converters.
+ *
+ * @author hylston.barbosa
+ */
+public class DateTimeUtils {
+
+  /**
+   * Converts an Instant into {@link ZonedDateTime}. The input should be an {@link Instant}. Defines
+   * the timezone as UTC.
+   *
+   * @author hylston.barbosa
+   */
+  public static ZonedDateTime fromDateToZonedDate(Date date) {
+
+    if (date != null) {
+      return date.toInstant().atZone(ZoneOffset.UTC);
```

Comment:
we can talk offline more, but this does not seem to be a replacement for the old code